bin/deploy.sh

```diff
@@ -30,21 +30,11 @@ source ./borrower/deploy.sh
 
 # finalize deployment
 message Finalize Deployment
-GOVERNANCE="0x$ETH_FROM"
 
 seth send $ROOT_CONTRACT 'prepare(address,address,address)' $LENDER_DEPLOYER $BORROWER_DEPLOYER $GOVERNANCE
```

Comment:
if you remove it this line would not work anymore?

Reply:
it will, I have added it to the local config